mitodl/bootcamp-ecommerce
# Payment drawer

#### What are the relevant tickets?
Fixes #572 

#### What's this PR do?
Adds `PaymentDisplay` to allow people to pay from the application page. Some API changes were also made to do this.

#### How should this be manually tested?
First, set up Cybersource environment variables. I can send you the variables for testing via keybase. Go to `/application` and expand one of the applications. Click 'Make a Payment'. Type in a number in the textbox and click 'Pay Now'. You should be directed to Cybersource.

Note: Once you complete a payment it will redirect to `/pay/`. I'm not sure what should happen at this point so I left the behavior as is.
